src/components/SearchFilter/SearchFilter.tsx

```diff
@@ -0,0 +1,229 @@
+      const queryArray = [...searchPageFilters]
+      queryArray.splice(index, 1)
+      setSearchPageFilters(queryArray)
+    } else {
+      setSearchPageFilters([...searchPageFilters, queryValue])
+    }
+  }
+
+  return (
+    <div className={styles.searchFilter}>
+      <div className={styles.headerContainer}>
+        <h3>Filter Search</h3>
+      </div>
+
+      <Grid col="auto">
+        <h4 className={styles.subHeader}>Categories:</h4>
+        <Checkbox
+          id="application-filter"
+          name="application"
+          label="Application"
+          value="application"
+          className={styles.checkbox}
+          checked={filterItems.application}
+          onChange={(e) => {
+            setFilterItems({
+              ...filterItems,
+              application: !filterItems.application,
+            })
+            updateCheckedItems(`category:${e.target.value}`)
+          }}
+        />
+        <Checkbox
+          id="news-filter"
+          name="news"
+          label="News"
+          value="news"
+          className={styles.checkbox}
+          checked={filterItems.news}
+          onChange={(e) => {
+            setFilterItems({
+              ...filterItems,
+              news: !filterItems.news,
+            })
+            updateCheckedItems(`category:${e.target.value}`)
+          }}
+        />
+        <Checkbox
+          id="documentation-filter"
+          name="documentation"
+          label="Documentation"
+          value="documentation"
+          className={styles.checkbox}
+          checked={filterItems.documentation}
+          onChange={(e) => {
+            setFilterItems({
+              ...filterItems,
+              documentation: !filterItems.documentation,
+            })
+            updateCheckedItems(`category:${e.target.value}`)
+          }}
+        />
+      </Grid>
+      <Grid col="auto">
+        <Label htmlFor="label-dropdown" className={styles.subHeader}>
+          Labels:
+        </Label>
+        <Dropdown
+          id="label-dropdown"
+          name="label-dropdown"
+          data-testid="label-dropdown"
+          value={filterItems.dropdown ? filterItems.dropdown : 'default'}
+          onChange={(e) => {
+            // Need to remove any previous label and add the newly selected one
+            const labelToAdd = e.target.value
+
+            // Filter out any previous label from the query
+            const filteredArray = searchPageFilters.filter(
+              (label) => !label.includes('label:')
+            )
+
+            // Add the new label to the array
+            filteredArray.push(`label:${labelToAdd}`)
+            setSearchPageFilters(filteredArray)
+
+            // Update the state of the dropdown
+            setFilterItems({
+              ...filterItems,
+              dropdown: labelToAdd,
+            })
+          }}>
+          <option value="default" disabled>
+            None applied
+          </option>
```

Comment:
Question: Should this option be disabled? I tried to reset the labels to none and had to click `Reset` which resets the entire form. I wonder if it would be fine to allow this to be selected.

Reply:
Good catch. It shouldn't be disabled.